### PR TITLE
modified dubbo-samples-nacos-configcenter for dubbo#8217

### PR DESCRIPTION
**Pull request by zrlw**
for apache/dubbo#8217

### CODE REVIEW

File: dubbo-samples-nacos/dubbo-samples-nacos-configcenter/src/main/resources/spring/configcenter-consumer.xml [hunk 1]
```diff
@@ -30,6 +30,4 @@
 
     <dubbo:config-center highest-priority="false" address="nacos://${nacos.address:127.0.0.1}:8848"/>
 
-    <dubbo:reference id="demoService" interface="org.apache.dubbo.samples.configcenter.api.DemoService"/>
-
 </beans>
```
Comment (AlbumenJ) [inline dubbo-samples-nacos/dubbo-samples-nacos-configcenter/src/main/resources/spring/configcenter-consumer.xml:33]: Please do not change this

Comment (zrlw) [reply dubbo-samples-nacos/dubbo-samples-nacos-configcenter/src/main/resources/spring/configcenter-consumer.xml]: reverted

File: dubbo-samples-nacos/dubbo-samples-nacos-configcenter/src/test/java/org/apache/dubbo/samples/configcenter/DemoServiceIT.java [hunk 2]
```diff
@@ -19,19 +19,19 @@
...
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(locations = "classpath:spring/configcenter-consumer.xml")
+@EnableDubbo
```
Comment (AlbumenJ) [inline dubbo-samples-nacos/dubbo-samples-nacos-configcenter/src/test/java/org/apache/dubbo/samples/configcenter/DemoServiceIT.java:32]: was it necessary to change this file?

Comment (zrlw) [reply dubbo-samples-nacos/dubbo-samples-nacos-configcenter/src/test/java/org/apache/dubbo/samples/configcenter/DemoServiceIT.java:32]: These changing is for regression test. if not changing,  a ClassCastException will not be trigged running with dubbo 2.7.12 or prior version.

**Comment by zrlw** [comment]
@kylixs

**Pull request merged**